data/categories.txt

```diff
@@ -15872,3 +15872,16 @@ uk:Колекціонер
 vi:Nhà sưu tập
 zh-Hans:收藏家
 zh-Hant:集電極
+
+man_made-cairn
+en:4Cairn
```

Comment:
Should it really be searchable? Who will guess that Cairn is supported by OM, and tries to type its name properly in the search?

Reply:
assuming cairns with names will still be searchable without a category type name, i'll remove these strings 👍️

Reply:
I'd keep them searchable. Cairns are quite often erected as major "milestones" e.g. on a mountain pass and e.g. in ex-USSR mountaineering groups traditionally leave there written notes to future adventurers. And they don't necessary have a name.

Reply:
@vng if objects are not added to categories.txt, are they searched by their name anyway?